scrapbook/models.py

```diff
@@ -24,6 +25,12 @@
 from .exceptions import ScrapbookException
 from .utils import kernel_required, deprecated
 
+if sys.version_info > (3, 0, 0):
```

Comment:
Better to do:

```
try:
    from urllib.parse import urlparse  # Py3
except:
    from urlparse import urlparse  # Py2
```

Reply:
Thanks for the info - I've patched the unit test as suggested.